node/auth/authenticator_test.go

```diff
@@ -321,117 +209,35 @@ func TestKeyExpiry(t *testing.T) {
 	require.NoError(t, err)
 	request.Signature = signature
 
-	err = authenticator.AuthenticateStoreChunksRequest(context.Background(), "localhost", request, start)
+	hash, err := authenticator.AuthenticateStoreChunksRequest(context.Background(), request, start)
+	require.NoError(t, err)
+	expectedHash, err := hashing.HashStoreChunksRequest(request)
 	require.NoError(t, err)
+	require.Equal(t, expectedHash, hash)
 
 	// Since time hasn't advanced, the authenticator shouldn't have fetched the key again
 	chainReader.Mock.AssertNumberOfCalls(t, "GetDisperserAddress", 1)
 
 	// Move time forward to just before the key expires.
 	now := start.Add(59 * time.Second)
-	err = authenticator.AuthenticateStoreChunksRequest(context.Background(), "localhost", request, now)
+	hash, err = authenticator.AuthenticateStoreChunksRequest(context.Background(), request, now)
 	require.NoError(t, err)
+	require.Equal(t, expectedHash, hash)
 
 	// The key should not yet have been fetched again.
 	chainReader.Mock.AssertNumberOfCalls(t, "GetDisperserAddress", 1)
 
 	// Move time forward to just after the key expires.
 	now = now.Add(2 * time.Second)
-	err = authenticator.AuthenticateStoreChunksRequest(context.Background(), "localhost", request, now)
+	hash, err = authenticator.AuthenticateStoreChunksRequest(context.Background(), request, now)
 	require.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
here is another duplicate `require.NoError(t, err)`

Reply:
fixed